tailwind.config.ts

```diff
@@ -31,6 +31,7 @@ export default {
         700: '#6E7B83',
         800: '#556066',
         900: '#2C3437',
+        950: '#0D121C',
```

Comment:
We should probably remove the black one 👇 below and update any CSS component that is using "black" (eg: `bg-black`) to `bg-neutral-950`

Reply:
I put the validation in PR description.
Later before merging, I'll rebase `main` and verify one more time.

Reply:
So just to double check, you found no usages of black so far?

Reply:
Let me rebase once and verify, I think the recent changes have `bg-black` usages.